### ISSUE TITLE
Add basic bob implementation

### ISSUE THREAD

**Issue by tehKaiN** [tehKaiN/ACE#46]
Currently, bob mechanism must be written from scratch - it should be for most games, but there should be some means to prototype things and I guess that most implementations could have a common ancestor.

It should be mentioned that blitter could be set up using CPU or Copper - the latter option ensures constant blitter throughput.

**Comment by SamuraiCrow** [comment]
Blitter queues run by the copper cause race conditions to exist and should not be used with any other copper usage except maybe a little initialization at the top of the screen.  If the blitter can't keep up with the screen refresh taking place at the end of the copper blit list it can crash the system with the copper writing to the blitter while it's still working.

**Comment by tehKaiN** [comment]
That's true and I've stumbled upon it the other day. I've done quite good CPU-driven bob manager for OpenFire, refined it in AMIner and I think I'll try to incorporate it to ACE when it's flexible/mature enough.

**Comment by tehKaiN** [comment]
the *bob_new* manager from openfire/aminer/germz/chaosarena/aminer is now mature enough to be incorporated to the core ACE codebase, I think. Will add PR soon!
